lib/matplotlib/backends/backend_pdf.py

```diff
@@ -2236,6 +2236,18 @@ def encode_string(self, s, fonttype):
             return s.encode('cp1252', 'replace')
         return s.encode('utf-16be', 'replace')
 
+    @staticmethod
+    def _font_supports_char(fonttype, char):
+        """
+        Returns True if the font is able to provided the char in a PDF
+
+        For a Type 3 font, this method returns True only for single-byte
+        chars.  For Type 42 fonts this method always returns True.
```

Comment:
Should this be explicit in the code and should we raise for other fonttypes?

Reply:
We already have rcParam checks that disallow other fonttypes..

Reply:
Yes, I've learned to be catious. If somebody implements a new fonttype, they may overlook this function. So, forcing future font types to make an explicit decision here is a plus (Unless you say that the default should generally be True and the Type-3 handling is a rare exception.

Reply:
At the moment, the function is only called in case of a Type 3 or 42, but I agree it's probably safer to raise a `NotImplementedError` (?) if the font type is neither.

Reply:
You could imagine an object-oriented solution where the different types of fonts are different classes, and this would be a method on the font. But that would likely be unnecessary overengineering. Perhaps the `NotImplementedError` is sufficient defence against future accidents.